FirebaseDynamicLinks/Sources/Public/FIRDynamicLinks.h

```diff
@@ -75,12 +90,12 @@ NS_SWIFT_NAME(DynamicLinks)
  * @return Dynamic Link object if the URL is valid and has link parameter, otherwise nil.
  */
 - (nullable FIRDynamicLink *)dynamicLinkFromUniversalLinkURL:(NSURL *)url
-    NS_SWIFT_NAME(dynamicLink(fromUniversalLink:));
+    NS_SWIFT_NAME(dynamicLink(fromUniversalLink:))
+        DEPRECATED_MSG_ATTRIBUTE("Use dynamicLinkFromUniversalLinkURL:completion: instead.");
```

Comment:
@ryanwilson @paulb777 Are there any implications to adding a deprecated message to a currently public API?

Reply:
@ncooke3 It should be fine if the API council agrees with the transition recommendation.